support/k8s/k8s-workload-registrar/config_crd.go

```diff
@@ -90,11 +102,41 @@ func (c *CRDMode) Run(ctx context.Context) error {
 	}
 
 	if c.WebhookEnabled {
+		// Backwards compatibility check
+		exists, err := c.certDirExistsAndReadOnly()
+		if err != nil {
+			return fmt.Errorf("checking webhook certificate directory permissions: %w", err)
+		}
+		if exists {
+			log.Warn("Detected statically mounted webhook certificate directory, support for this will be removed in a future version. " +
+				"Refer to README for instructions on using SPIRE Server to populate webhook certificates.")
+		} else {
+			webhookSVID, err := webhook.NewSVID(ctx, webhook.SVIDConfig{
+				Cluster:            c.Cluster,
+				S:                  svidClient,
+				Log:                log,
+				Namespace:          myPodNamespace,
+				TrustDomain:        c.TrustDomain,
+				WebhookCertDir:     c.WebhookCertDir,
+				WebhookServiceName: c.WebhookServiceName,
+			})
+			if err != nil {
+				return err
+			}
+			if err = webhookSVID.MintSVID(ctx, nil); err != nil {
+				return err
+			}
+			go func() {
+				if err := webhookSVID.SVIDRotator(ctx); err != nil {
+					log.Fatalf("failed rotating webhook certificate: %v", err)
```

Comment:
It seems problematic to take down the registrar on a transient failure to mint (which is totally a possibility depending on the keymanager in use). I'd recommend not cycling the registrar until the webhook certificate is in immediate danger of becoming expired (or has expired).